sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/communication/Message.groovy

```diff
@@ -0,0 +1,9 @@
+package life.qbic.portal.sampletracking.communication
+
+/**
+ * <p>A message to be sent around. This class should be extended to hold relevant content.</p>
+ *
+ * @since 1.0.0
+ */
+abstract class Message {
+}
```

Comment:
What's the benefit to use this class here? This class provides no functionality other than grouping classes. What was your idea in introducing it?